components/Document/api/fetchPostFromS3.ts

```diff
@@ -15,6 +16,14 @@ const fetchPostFromS3 = async ({ s3Url, cleanIntroEmptyContent = true }: Props):
       });
     }
     let _html = await response.text();
+    _html = sanitizeHtml(_html, {
```

Comment:
Nit: Could use `then()` here and format and order list of attributes.

```
    let _html = await response.text().then((text) =>
      sanitizeHtml(text, {
        allowedTags: sanitizeHtml.defaults.allowedTags.concat([
          "source",
          "video",
        ]),
        allowedAttributes: {
          ...sanitizeHtml.defaults.allowedAttributes,
          video: [
            "autoplay",
            "controls",
            "height",
            "loop",
            "muted",
            "src",
            "type",
            "width",
          ],
          source: ["src", "type"],
        },
      })
    );
```

Reply:
Ended up reverting to avoid deep nesting as suggested by Kobe.